spec/game_team_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "spec_helper"
+
+RSpec.describe GameTeam do
+  before(:each) do
+    gameteam_data = {
+        game_id: "2012030221",
+        team_id: "3",
+        hoa: "away",
+        result: "LOSS",
+        settled_in: "OT",
+        head_coach: "John Tortorella",
+        goals: 2,
+        shots: 8,
+        tackles: 44,
+        pim: 8,
+        ppo: 3,
+        ppg: 0,
+        faceoff_win_per: 44.8,
+        giveaways: 17,
+        takeaways: 7
+    }
+    @gameteam_attr_test = GameTeam.new(gameteam_data)
+end
+
+describe "initialize" do
+    it "exists" do
+        expect(@gameteam_attr_test).to be_an_instance_of(GameTeam)
+    end
+
+    it "has readable attributes" do
```

Comment:
I like these tests too, in my class init I only tested for the data type so far but I think it's good to add testing to make sure we're getting the exact data we want as well like you did here